pkg/storaged/client.js

```diff
@@ -1030,7 +1030,7 @@ function stratis2_start() {
                             .input(passphrase);
                 };
 
-                client.stratis_unlock_pool = (uuid) => {
+                client.stratis_start_pool = (uuid) => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18679-20230424-131415-eca64e0e-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)